infrastructure/cdn-in-a-box/traffic_ops/trafficops-init.sh

```diff
@@ -71,6 +71,21 @@ delayfor() {
     local d="${f%/*}"
 
     case $d in
+        deliveryservices)
+            local cdns_json
+            cdns_json="$(to-get "api/${TO_API_VERSION}/cdns?name=${CDN_NAME}")"
+            if [[ "$(<<<"$cdns_json" jq '.response[].dnssecEnabled')" == false ]]; then
+              echo "DNSSEC is not enabled for ${CDN_FQDN}"
+              return
+            fi
+            jq_filter='.date = $date | .key = $cdn_name | .name = $domain | .ttl = 60 | .kskExpirationDays = 365 | .zskExpirationDays = 30'
+            generate_dnssec_body="$(jq -n --arg cdn_name "$CDN_NAME" --arg domain "$CDN_FQDN" --arg date "$(date --rfc-3339=seconds)" "$jq_filter")"
```

Comment:
Is this body missing the `dnssecEnabled` field? Won't it default to false? After building CiaB i'm still seeing it as false, wondering if its cause of this?

Reply:
This is the body for `POST /cdns/dnsseckeys/generate`. The problem seems to be that `dnssecEnabled` is hard-coded to false in the postinstall script where the CDN being inserted.

https://github.com/apache/trafficcontrol/blob/4b25f4962ac32b398121b8fd3ccf4bcf97064e23/traffic_ops/install/bin/_postinstall.py#L1138-L1140

So we'll need a flag for that